templates/federated_reporting/cfsecret.py

```diff
@@ -0,0 +1,23 @@
+#!/var/cfengine/bin/cfengine-selected-python
```

Comment:
This should not have a shebang at all, it's not supposed to be run as script.

Reply:
sure.